kola/tests/sysext/disable_docker.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 CoreOS, Inc.
+// Copyright 2023 The Flatcar Maintainers.
```

Comment:
Let's keep both lines

Reply:
Git wrongly assumes I've copied and modified that file from `torcx/torcx.go`. That's incorrect, I copied `torcx.go` to `enable_docker.go` and created this file - `disable_docker.go` from scratch.

It would not make any sense for CoreOS to have written anything of this test as it is purely sysext specific.